torch/serialization.py

```diff
@@ -389,13 +389,20 @@ def persistent_load(saved_id):
         else:
             raise RuntimeError("Unknown saved id type: %s" % saved_id[0])
 
-    # try the legacy loader first, which only works if f is a tarfile
+    # on windows this can return invalid values
```

Comment:
What kind of invalid values? When does it return invalid values?

Reply:
right, I see. we know we're in binary mode so this point is mute. it's for unix file endings in text mode that this applies